libraries/aws/provisioning/src/aws_iot_provisioning_serializer.c

```diff
@@ -44,6 +44,22 @@
     #error "AWS_IOT_PROVISIONING_ENABLE_ASSERTS must be 0 or 1."
 #endif
 
+/*------------------------------------------------------------------*/
+
+/**
+ * @brief Represents the number of map entries in the request payload of the
+ * MQTT CreateCertificateFromCsr service API.
+ */
+static const size_t _numOfCreateKeysAndCertPayloadMapEntries = 0u;
+
+/**
+ * @brief Represents the number of map entries in the request
+ * payload of the MQTT CreateCertificateFromCsr service API.
+ * The ONLY payload entry is for the Certificate-Signing Request string
+ */
+static const size_t _numOfCertFromCsrPayloadMapEntries = 1u;
```

Comment:
Is there a reason these are `static consts` and not macros, or only preference? Also, these names seem a bit long, can they be shorter? Like `_numCsrCertMapEntries`

Reply:
No major reason besides the type-checking benefit brought with constants.
Shortened the name